src/framework/components/sound/component.js

```diff
@@ -414,6 +414,68 @@ class SoundComponent extends Component {
         return this._slots[name];
     }
 
+    /**
+     * Return a property from the slot with the specified name.
+     *
+     * @param {string} name - The name of the {@link SoundSlot} to look for.
+     * @param {string} property - The name of the property to look for.
+     * @returns {*} The value from the looked property inside the slot with specified name. May be undefined if slot does not exist.
```

Comment:
the type could be `boolean | undefined` instaed of '*'?

Reply:
or even better, the function should return false in that case .. never undefined? That would simplify other functions that call this function.

Reply:
ok fair enough, lets keep it generic then.